Bot.py

```diff
@@ -7,10 +7,12 @@
 import tweepy
 from tweepy.error import TweepError
 
+from cachetools import TTLCache, ttl

```

Comment:
Is this second import needed? I think it's only used for the keyword arg on L44, which doesn't require an import.

Reply:
I think VS Code auto inserted this at some point oops